ch_util/tools.py

```diff
@@ -1229,6 +1231,79 @@ def get_crate_channel(slot, sma):
     return default
 
 
+def get_default_frequency_map_stream() -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
```suggestion
def get_default_frequency_map_stream() -> tuple[np.ndarray, np.ndarray]:
```
or change the other annotations to Tuple